sdk/tables/data-tables/src/generated/models/mappers.ts

```diff
@@ -6,9 +6,9 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
```

Comment:
I wonder, are these changes by hand or do we have support for `core-https` in the code generator?

Reply:
@joheredi is going to help with updating the generator. 😄